docs/content/1.index.md

```diff
@@ -15,65 +15,68 @@ secondary:
   - https://github.com/nuxt/content
 snippet: yarn add --dev @nuxt/content
 ---
-
 #title
-Content made easy for developers
-
+Content made easy  
+for developers
 #description
-Nuxt Content reads the `content/` directory in your project, parses `.md`, `.yml`, `.csv` or `.json` files and creates a powerful data layer for your application. Bonus, use Vue components in Markdown with the MDC syntax.
+Write pages in markdown, use Vue components and enjoy  
+the power of Nuxt with a blazing fast developer experience.
 ::
 
-::card-grid
-#title
-Powerful Features
-
-#default
-  ::card
-  ---
-  icon: IconNuxt
-  title: Built for Nuxt 3
-  description: 'Take advantage of Nuxt 3 features: Vue 3, Auto-imports, Vite and Nitro server.'
-  ---
-  ::
-  
-  ::card
-  ---
-  icon: IconCopy
-  description: Write your content in mardkdown, yml, csv or json and query it in your components.
-  title: File-based CMS
-  ---
-  ::
-
-
-  ::card
+::n-section
+  ::n-hero
   ---
-  icon: IconMarkdown
-  title: MDC Syntax
-  description: Use your Vue components in Markdown files, supporting props, slots and nested components.
+  tag: Features
+  icon: heroicons-solid:sparkles
   ---
+  #title
+  Store your Content at  
+  the right place.
+  #description
+  Static content should be store in files instead of databases. No need to learn a new query language or complicated API docs to read. Fetch your content as data and display it directly in your components.
```

Comment:
```suggestion
  Static content should be stored in files instead of databases. No need to learn a new query language or read complicated API docs. Fetch your content as data and display it directly in your components.
```